### DISCUSSION TITLE
当审核/执行状态为warning也会自动审核不通过吗

### DISCUSSION THREAD

**Discussion by magicxiexiaodong** [hhyo/Archery#2437] [Q&A]
![image](https://github.com/hhyo/Archery/assets/46348633/4477ac3c-e1b1-4721-86a7-efd82ab4c502)

我提交了一个SQL审核，审核执行信息有warning级别，然后它就自动审核不通过了，这里在哪里可以配置warning是可以通过的

**Comment by magicxiexiaodong** [comment]
我理解了一些goinception的知识，看到了其实是在 inc_level 里面配置0 1 2 想知道这个怎么调整配置的呢

**Comment by ShelwinSu** [comment]
登录Archery后，进入配置项管理，选择配置项：系统设置，在功能模块配置下的：SQL上线里面设置：AUTO_REVIEW_WRONG项的值：1：表示SQL上线审核出现警告信息就驳回，2和空表示出现错误才驳回，其他设置表示不驳回。这里改成2，应该就可以了。
